### DISCUSSION TITLE
Upcoming Bluetooth LE Support?

### DISCUSSION THREAD

**Discussion by Brennnesselsmoker** [ndeadly/MissionControl#733] [Q&A]
Hey There devs,
I was wondering how I could connect my Series X Controller for several Months now and I wanted to ask for maybe an Update about how the Support is going or if it is not coming for a long time. I don't know how complicated this is but maybe an Update would be nice about the status quo :)
Thanks in advance and thanks for your service :)

**Comment by ndeadly** [comment]
It doesn't really make sense to give updates on it. I am working on it when I can, in my free time. A lot of that time is spent reverse engineering code and learning how things work through trial and error with the hardware. It's difficult to quantify progress when you don't have a clear picture of what the intermediate goals are. Nor does the average person care about random findings within the bluetooth service. When something noteworthy happens, you will hear about it.

If you're interested in following along, I post test builds and engage in discussion on the topic on my discord server.

**Reply by Brennnesselsmoker** [reply]
Okay, thanks for your fast answer and good luck!

**Reply by Brennnesselsmoker** [reply]
Hey there, are there any mention worthy updates on it?

**Reply by ndeadly** [reply]
There have certainly been some updates since 2023. Mostly in the way of supported controllers and stability. Most common controllers can be connected and used in some sense. There's still work to be done, but I think I've found solutions for most of the major problems. My goal is to try and get it done before the release of the Switch 2, but time and motivation has been scarce lately.
